src/components/Hero/hero.scss

```diff
@@ -0,0 +1,131 @@
+@import '../../styles/variables';
+
+.hero__container{
+    height: 765px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+
+    background-image: url(./backgroud-min.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: auto 765px;
+
+    @media screen and (min-width:($max-width-tablet)) {        
+        height: 540px;
+        background-size: cover;
+    }
+    @media screen and (min-width:($max-width-desktop)) {
+        height: 720px;
+        background-size: auto 720px;
+        }
+}
+
+.hero__tumb{
+    width: 100%;
+    color: $white;
+    text-align: center;
+    padding-top: 180px;
+    padding-bottom: 180px;
+    padding-left: 16px;
+    padding-right:16px;
+    margin-left: auto;
+    margin-right: auto;    
+    @media screen and (min-width:($max-width-mobile)) {
+        width: $max-width-mobile;     
+    }
```

Comment:
change all elements where you use padding and margin and use the shortest form

https://developer.mozilla.org/en-US/docs/Web/CSS/margin